### ISSUE TITLE
update copy

### ISSUE THREAD

**Issue by CatherineKlauss** [Qiskit/qiskit.org#280]
user feedback on copy: 

>> IBM Q awards copy needs to have a clear CTA 
>> Coding with Qiskit copy should not name someone that users do not recognize
>> IQX section needs copy that differentiates between Qiskit and IQX. 
      > From Ali: "IQX is a cloud service. Qiskit is a software that uses this cloud service."

**Comment by CatherineKlauss** [comment]
Also need to update header copy. Currently, "welcome to quantum" is cheesy/means nothing to current quantum researchers, and subtitle copy is too wordy, "what does computing software development framework mean?" . And rather than say "applications in research, education and business", we should show this in different section of the webpage. 

possible solution:
Qiskit
an open-source quantum computing software

**Comment by lerongil** [comment]
Can we separate some of these into different issues. Some of them are quick fixes i.e. removing names from coding with qiskit while others may require additional discussions.

**Comment by CatherineKlauss** [comment]
can do. the small fixes I'll put down as size:s, and the other ones I'll start with a suggested copy change and label as discussion.
